docs/security-considerations.rst

```diff
@@ -139,17 +146,26 @@ Sending and Receiving Ether
      means for the recipient to block progress in the sending contract. Again, the best practice here is to use
      a :ref:`"withdraw" pattern instead of a "send" pattern <withdrawal_pattern>`.
 
-Callstack Depth
+
+.. class:: strike
+Callstack Depth [not a vulnerability]
 ===============
 
-External function calls can fail any time because they exceed the maximum
-call stack of 1024. In such situations, Solidity throws an exception.
-Malicious actors might be able to force the call stack to a high value
-before they interact with your contract.
+On October 18th 2016 a hard fork occurred due to EIP 150, where the stack size limit is no more a vulnerability, thus call depth attacks are not exploitable.
```

Comment:
If you keep the paragraph like that, the reader will not know what a "callstack depth attack" should be. Also, please break the lines after around 80 to 100 characters to aid readability in github.